### PR TITLE
LPD-30049 Disable batch engine bundles on upgrade

### PR DESCRIPTION
**Pull request by 4lejandrito**
https://liferay.atlassian.net/browse/LPD-30049.

### CODE REVIEW

File: modules/apps/batch-engine/batch-engine-test/src/testIntegration/java/com/liferay/batch/engine/internal/test/BatchEngineBundleTrackerTest.java [hunk 1]
```diff
@@ -69,11 +76,25 @@ public class BatchEngineBundleTrackerTest {
 	public static final AggregateTestRule aggregateTestRule =
 		new LiferayIntegrationTestRule();
 
+	@BeforeClass
+	public static void setUpClass() {
+		_originalStopWatch = ReflectionTestUtil.getFieldValue(
```
Comment (ccorreagg) [inline modules/apps/batch-engine/batch-engine-test/src/testIntegration/java/com/liferay/batch/engine/internal/test/BatchEngineBundleTrackerTest.java:81]: I think we could use instead:

```
_originalStopWatch = ReflectionTestUtil.getAndSetFieldValue(
	DBUpgrader.class, "_stopWatch", null);
```

**Review by ccorreagg — COMMENTED** [review]
Just a comment

**Comment by ccorreagg** [comment]
ci:test:sf

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by ccorreagg** [comment]
ci:test:relevant

**Comment by ccorreagg** [comment]
ci:forward

File: modules/apps/batch-engine/batch-engine-test/src/testIntegration/java/com/liferay/batch/engine/internal/test/BatchEngineBundleTrackerTest.java [hunk 2]
```diff
@@ -69,6 +76,18 @@ public class BatchEngineBundleTrackerTest {
 	public static final AggregateTestRule aggregateTestRule =
 		new LiferayIntegrationTestRule();
 
+	@BeforeClass
+	public static void setUpClass() {
+		_originalStopWatch = ReflectionTestUtil.getAndSetFieldValue(
```
Comment (4lejandrito) [inline modules/apps/batch-engine/batch-engine-test/src/testIntegration/java/com/liferay/batch/engine/internal/test/BatchEngineBundleTrackerTest.java:81]: @ccorreagg maybe we can apply this to other tests that followed the same pattern?

Comment (ccorreagg) [reply modules/apps/batch-engine/batch-engine-test/src/testIntegration/java/com/liferay/batch/engine/internal/test/BatchEngineBundleTrackerTest.java:81]: Yes, I will create a new PR for it

**Pull request closed without merging**